properties.schema

```diff
@@ -208,6 +208,15 @@
                     }
                   }
                 },
+                "_suppressMarking": {
+                  "type":"boolean",
+                  "required":false,
+                  "title": "Suppress marking until assessment completion",
+                  "inputType": { "type": "Boolean", "options": [false, true]},
+                  "default": true,
+                  "validators": [],
+                  "help": "If enabled, individual question marking will only occur when the user has completed the assessment. Dependant on Show Marking being enabled."
```

Comment:
I think "dependant" should actually be "dependent" here.  Can you put this?
"Note: this is dependent on 'Show Marking' being enabled on the question component."